RasterPropMonitor/Core/MonitorPage.cs

```diff
@@ -576,11 +577,12 @@ public void RenderBackground(RenderTexture screen)
                     }
                     break;
                 case BackgroundType.Texture:
+                    //call clear before redraw of textures
                     GL.Clear(true, true, ourMonitor.emptyColorValue);
                     Graphics.DrawTexture(new Rect(0, 0, screen.width, screen.height), backgroundTexture);
                     break;
                 case BackgroundType.Handler:
-
+                    //No clear here as it would interfere with the handlers(Causing effects such as VesselViewer to blink)
```

Comment:
The problem is the GL.Clear here - VV assumes that RPM doesn't clear the screen every frame, which was true prior to the change I made.  So when it's not refreshing every frame, we get some ugliness.  Your change with the tweak I suggested does fix the problem without forcing VV to update, and I haven't seen any other problems yet (although I don't have Engine Ignitor, and I haven't had a chance to test with the new DPAI).